coordinator/statement_executor.go

```diff
@@ -1166,3 +1172,57 @@ func joinUint64(a []uint64) string {
 	}
 	return buf.String()
 }
+
+// fmtDuration formats a duration to a string without dot(.) and unnecessary '0's.
+func fmtDuration(td time.Duration) string {
```

Comment:
Can you use `influxql.FormatDuration` instead? Thanks.

Reply:
sure. Please review again.